client/package.json

```diff
@@ -20,17 +20,17 @@
     "test-silently": "npm run test -- --watch=false --no-progress --browsers=jsdom",
     "test-live": "npm run test -- --watch=true --browsers=Chromium",
     "stats": "webpack-bundle-analyzer dist/client/stats.json",
-    "lint": "ng lint",
-    "lint-write": "ng lint --fix",
+    "lint": "ng lint && npm run prettify-check",
+    "lint-write": "npm run prettify-write && ng lint --fix",
```

Comment:
The order "linting -> formatting" was on purpose and should be kept: The automatic fixes from the linter are not always compatible with prettiert. So, linting has to run first, and prettier afterwards. To circumvent the issue that you can't see the output, the solution which you deleted was implemented (but maybe did not work correctly lately).

Also, I don't think `npm run lint` should execute the prettify commands.